archlinuxcn/fractal-git/PKGBUILD

```diff
@@ -0,0 +1,37 @@
+# Maintainer: xiretza <xiretza+aur@xiretza.xyz>
+# Contributor: Falk Alexander Seidl <fa@terminal.run>
+
+pkgname=fractal-git
+_gitname=fractal
+epoch=1
+pkgver=6.r150.g462b5cd5
+pkgrel=1
+pkgdesc="Matrix messaging app for GNOME written in Rust"
+arch=('i686' 'x86_64')
+license=('GPL3')
```

Comment:
- ```license=('GPL3')``` should be replaced with ```license=('GPL-3.0-only')``` according to [SPDX License Identifiers](https://gitlab.archlinux.org/archlinux/rfcs/-/blob/master/rfcs/0016-spdx-license-identifiers.rst).